src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,237 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> last;
+    private Node<T> currentNode;
+    private Node<T> nextNode = null;
+    private Node<T> prevNode = null;
+    private int index;
+
+    public MyLinkedList() {
+        head = null;
+        last = null;
+    }
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(T item) {
+            next = null;
+            this.item = item;
+            prev = null;
```

Comment:
```suggestion
            this.item = item;
```
class fields have default values
if it is a reference type the default value is null